usaspending_api/agencies/management/commands/load_agencies_sql/clean_up.sql

```diff
@@ -0,0 +1,4 @@
+drop table if exists temp_load_agencies_agency;
+drop table if exists temp_load_agencies_raw_agency;
+drop table if exists temp_load_agencies_subtier_agency;
+drop table if exists temp_load_agencies_toptier_agency;
```

Comment:
This will fail silently if I forget to add a temp table here. It would be nice if it was programmatically pulling it off of a list that was populated every time a sql file made a temp table.

Reply:
Good news!  It didn't dawn on me at the time (probably because one of the earliest versions didn't run in a single transaction), but because the entire process runs in a single transaction, I can use true temporary tables.  No explicit cleanup required!